src/house_package_saachi/utils.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import StratifiedShuffleSplit
+
+def stratified_split(df, strat_col, test_size=0.2, random_state=42):
+    split = StratifiedShuffleSplit(
+        n_splits=1, test_size=test_size, random_state=random_state
+    )
+    for train_index, test_index in split.split(df, df[strat_col]):
+        strat_train_set = df.loc[train_index]
+        strat_test_set = df.loc[test_index]
+    return strat_train_set, strat_test_set
```

Comment:
you can have this function in `data_preprocessing.py` instead of a separate `utils.py`

Reply:
Removed utils.py file and added its content in data_preprocessing.py file